specs/MultiProfile.md

```diff
@@ -169,20 +169,37 @@ void ScenarioCookieManagement::DeleteAllCookies()
 ### Delete profile
 
 ```cpp
-HRESULT AppWindow::DeleteProfile(ICoreWebView2Controller* controller)
+HRESULT AppWindow::DeleteProfile(ICoreWebView2* webView2)
 {
-    wil::com_ptr<ICoreWebView2> coreWebView2;
-    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
-    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
-    if (webview7)
+    DCHECK(webView2);
+    wil::com_ptr<ICoreWebView2Profile> profile;
+    CHECK_FAILURE(webView2->get_Profile(&profile));
```

Comment:
`get_Profile` is a method on `ICoreWebView2_13`, so we need to

```cpp
auto webview13 = wil::try_com_query<ICoreWebView2_13>(webview2);
if (webview13)
{
    CHECK_FAILURE(webview13->get_Profile(&profile));
}

Reply:
ok, will change to it.